nat/traversal/events_tracker.go

```diff
@@ -62,23 +53,26 @@ func NewEventsTracker() *EventsTracker {
 func (et *EventsTracker) ConsumeNATEvent(event Event) {
 	log.Info(eventsTrackerLogPrefix, "got NAT event: ", event)
 
-	et.lastEvent = event
+	et.lastEvent = &event
 	select {
 	case et.eventChan <- event:
 	case <-time.After(300 * time.Millisecond):
 	}
 }
 
-// LastEvent returns the last known event
-func (et *EventsTracker) LastEvent() Event {
+// LastEvent returns the last known event and boolean flag, indicating if such event exists
+func (et *EventsTracker) LastEvent() (Event, bool) {
```

Comment:
We can just return a pointer to `Event`, in this case `nil` could be a valid value and we don't need a `bool` here.

Reply:
I did like that at first, but `bool` seemed to clarify more that value can be missing. But I can change this back :)

Reply:
Fixed